Sources/PointFree/Mailgun/SendEmail.swift

```diff
@@ -0,0 +1,56 @@
+import Either
+import Html
+import Prelude
+
+func sendEmail(
+  from: String,
+  to: [String],
+  subject: String,
+  content: Either3<String, [Node], (String, [Node])>,
+  domain: String = "mg.pointfree.co"
+  )
+  -> EitherIO<Unit, SendEmailResponse> {
+
+    let (plain, html): (String, String?) =
+      destructure(
```

Comment:
we can provide a `destructure` on `Either3` for allowing us to handle each of the 3 cases!